### ISSUE TITLE
Upload error

### ISSUE THREAD

**Issue by slecrom** [ARTbio/AnalyseGenome#36]
I tried to upload my first dataset. And I got en error message from the Galaxy server. However, I followed the tutorial as last year : https://artbio.github.io/startbio/AnalyseGenomes_2023/Loading_data_in_galaxy/#1-upload-of-reference-files-as-a-batch-of-multiple-urls-programmatic-file-naming

Here are the screenshot of what I have done and what I got.

![Capture d’écran 2024-10-31 à 13 24 10](https://github.com/user-attachments/assets/757b518a-5e1a-4d5a-acdc-df41ca7f4569)
![Capture d’écran 2024-10-31 à 13 33 34](https://github.com/user-attachments/assets/55ec82ab-bfef-4139-98d7-28b04565daaf)
![Capture d’écran 2024-10-31 à 13 34 02](https://github.com/user-attachments/assets/dfad3763-36d0-49a8-bd9c-4db62eeb1c3f)
![Capture d’écran 2024-10-31 à 13 34 16](https://github.com/user-attachments/assets/226da013-ea42-4cf8-99e5-9706f4026228)
![Capture d’écran 2024-10-31 à 13 34 37](https://github.com/user-attachments/assets/18ef0024-183c-448c-a693-64a1b1a41ccf)

**Comment by slecrom** [comment]
I tested the import on mississipi[2] and it works

**Comment by slecrom** [comment]
I created a new VM built with the 23.1 release of Galaxy from the main branch and during the import of the reference file I got  an error too. But not the same one.

![Capture d’écran 2024-10-31 à 15 45 27](https://github.com/user-attachments/assets/bbbe7136-8e06-4f00-9d43-e279bce60ccc)

**Comment by drosofff** [comment]
Yes @slecrom. At first glance, this is linked to the release_24.01. I'll have a look this weekend. The first obvious thing to test is release_24.0, or release_23.2.
There is a (reproductible) issue with sqlite version. The alternate solution may be to read the release notes and adapt the sh deployment script to a potential new feature of the latest Galaxy releases.
Of note, I did not know this error because our Galaxy servers are always relying on postgresql, not sqlite.

**Comment by drosofff** [comment]
After investigation, I can confirm that this is an issue from the Galaxy code that relies on python versions with appropriate SQLite versions. I am currently testing the trivial fix: changing the OS from Ubuntu 20.04 to Ubuntu 22.04

**Comment by drosofff** [comment]
@slecrom Yes, the simplest way to fix this bug is to switch this year to Ubuntu 22.04. I  have updated the Startbio sections accordingly.

**Comment by slecrom** [comment]
Thank you for your help. I tested the installation using Ubuntu 22.04 and the deploy_galaxy.sh script located inside the ag-2024 branch and uploading datasets is working now.
